### ISSUE TITLE
Applet Bug

### ISSUE THREAD

**Issue by djonespm** [linuxmint/cinnamon-spices-applets#6676]
### Applet version/Build date

3.6.4 (2024-11-23 13:11:41)

### Cinnamon version

3.2

### Distribution

mint 23.2

### Applet name and maintainer

weather@mockturtl @Gr3q

### What happened?

Service error.
I have  no idea what service is in error or how to go about correcting / reconnecting / reinstalling or whatever is necessary to correct this. I am set up for MET Norway as the provider. As far as I can recall, I have made no changes to the configuration or settings for this applet.

**Comment by djonespm** [comment]
Folks, this problem has resolved and the applet is working as expected. All that I did was go into the configure area and click on the tab to create a file of the problem, thinking that should have been done and attached when I set up this report. I noticed that immediately the tray icon cleared of the service error and seems to be reporting correctly. Seems strange to me that these issues would be connected, but that's what I saw. This BUG report can be dismissed / closed oar whatever.
